lib/operations/common/repoUtils.ts

```diff
@@ -36,7 +36,7 @@ export async function doWithAllRepos<R, P>(ctx: HandlerContext,
     const promises = ids.map(id =>
         () => repoLoader(id)
             .catch(err => {
-                logger.warn("Unable to load repo %s/%s: %s", id.owner, id.repo, err);
+                logger.debug("Unable to load repo %s/%s: %s", id.owner, id.repo, err);
```

Comment:
I've noticed a few places where failures that were logged as warnings are now debug. I presume you have your reasons…